src/aics-image-viewer/components/App/index.tsx

```diff
@@ -103,6 +106,8 @@ const defaultProps: AppProps = {
     showBoundingBox: false,
     view: "3D", // "XY", "XZ", "YZ"
     mode: "default", // "pathtrace", "maxprojection"
+    backgroundColor: BACKGROUND_COLOR as [number, number, number],
+    boundingBoxColor: BOUNDING_BOX_COLOR as [number, number, number],
```

Comment:
i suggest moving the [number,number,number] typing over to where the variable is defined rather than here.  you might have to rename the file to be a ts rather than a js file?

Reply:
we can be gradually walking this codebase from js to ts overall